Sources/Purchasing/TransactionsFactory.swift

```diff
@@ -18,44 +18,14 @@ enum TransactionsFactory {
 
     static func nonSubscriptionTransactions(
         withSubscriptionsData subscriptionsData: [String: [CustomerInfoResponse.Transaction]]
-    ) -> [StoreTransaction] {
+    ) -> [NonSubscriptionTransaction] {
         subscriptionsData
-            .flatMap { (productID, transactions) -> [StoreTransaction] in
+            .flatMap { (productID, transactions) -> [NonSubscriptionTransaction] in
                 transactions
                     .lazy
-                    .compactMap { BackendParsedTransaction(with: $0, productID: productID) }
-                    .map { StoreTransaction($0) }
+                    .compactMap { NonSubscriptionTransaction(with: $0, productID: productID) }
             }
             .sorted { $0.purchaseDate < $1.purchaseDate }
     }
 
 }
-
-/// `StoreTransactionType` backed by data parsed from the server
-private struct BackendParsedTransaction: StoreTransactionType {
-
-    let productIdentifier: String
-    let purchaseDate: Date
-    let transactionIdentifier: String
-    let quantity: Int
```

Comment:
Would it make sense to add this property to the new model? I see it's been set to 1, so I'm not sure it's coming from the backend

Reply:
It wasn't. We only had it so we could expose the SK2 version, but since it was a dummy value I thought not to until the backend sends it.